### PR TITLE
Fix backend

### PR DESCRIPTION
**Pull request by Dash293**

### CODE REVIEW

**Review by ThomasDCW — DISMISSED** [review]
ok

File: backend/src/models/FavoriteManager.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ class FavoriteManager extends AbstractManager {
 
   findAllFavorites(id) {
     return this.connection.query(
-      `select lesson.* from user INNER JOIN userFavLesson ON user.id=userFavLesson.user_id INNER JOIN lesson ON userFavLesson.lesson_id=lesson.id where user.id = ?`,
+      `select lesson.id, lesson.fileName, lesson.schoolTopic_id, lesson.title, lesson.subTitle, lesson.musicStyle, lesson.fileLocation, lesson.lyrics, lesson.img, lesson.duration, lesson.schoolClass_id from user INNER JOIN userFavLesson ON user.id=userFavLesson.user_id INNER JOIN lesson ON userFavLesson.lesson_id=lesson.id where user.id = 1
```
Comment (LuigiMenez) [inline backend/src/models/FavoriteManager.js:15]: Are you still looking for the data of user id 1?

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix minor

**Review by ThomasDCW — APPROVED** [review]
ok for me

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**